packages/cactus-plugin-satp-hermes/src/knex/knexfile.ts

```diff
@@ -1,16 +1,34 @@
 import path from "path";
 import { v4 as uuidv4 } from "uuid";
+import dotenv from "dotenv";
+
+dotenv.config({ path: path.resolve(__dirname, "../../.env") });
```

Comment:
@Yogesh01000100 @RafaelAPB I recommend making this path configurable because once you compile the TS sources the depth of the directory structure will be different (plus there's a million other scenarios where users would want to customize where their .env file is located)

Reply:
@petermetz thank you for your review. @Yogesh01000100 please address @petermetz 's suggestion. Rest LGTM